lib/block_view_association_test.go

```diff
@@ -982,7 +1035,7 @@ func _testAssociations(t *testing.T, flushToDB bool) {
 			require.Equal(t, userAssociationEntries[0].TargetUserPKID, m3PKID)
 		} else {
 			require.Error(t, err)
-			require.Contains(t, err.Error(), "invalid query params")
+			require.Contains(t, err.Error(), "invalid query params: missing AssociationType")
```

Comment:
Maybe it's better for us to define some string consts for these? I mean I guess you've already updated the tests, but if we were to change them again, it would be annoying. Maybe if we update the errors again, then we start defining them as consts. and these consts could live in `block_view_association.go` instead of `constants.go` since it's only really necessary to support these tests.